.github/workflows/ci.yml

```diff
@@ -0,0 +1,194 @@
+---
+name: "CI"
+on:  # yamllint disable
+  - "push"
+  - "pull_request"
+
+jobs:
+  black:
+    runs-on: "ubuntu-20.04"
+    env:
+      INVOKE_NAUTOBOT_LOCAL: "True"
+    steps:
+      - name: "Check out repository code"
+        uses: "actions/checkout@v2"
+      - name: "Setup environment"
+        uses: "networktocode/gh-action-setup-poetry-environment@v1"
+      - name: "Linting: black"
+        run: "poetry run invoke black"
```

Comment:
I think it is better to keep it separate. 1) You can quickly see where the issue is, 2) the execution is faster because it runs in parallel vs sequentially with `invoke tests --lint-only`.